app/models/admin.rb

```diff
@@ -7,4 +7,36 @@ class Admin < User
   after_destroy :revoke_repository_permissions
   after_update  :maintain_repository_permissions
 
+  # for an admin, we just want all the assignment groupings submitted
+  def get_num_assigned(assignment)
+    assignment.groupings.size
+  end
+
+  # for an admin, we want all the assignment groupings marked completed
+  def get_num_marked(assignment)
+    n = assignment.groupings.count(marking_completed: true)
+    n
+  end
+
+  def get_num_annotations(assignment)
+    n = 0
+    assignment.groupings.each do |x|
```

Comment:
You should be able to use the [sum](http://apidock.com/rails/Enumerable/sum) method here.